dolphinscheduler-bom/pom.xml

```diff
@@ -56,7 +56,7 @@
         <hadoop.version>3.2.4</hadoop.version>
         <cron-utils.version>9.1.6</cron-utils.version>
         <h2.version>2.2.220</h2.version>
-        <mysql-connector.version>8.0.16</mysql-connector.version>
+        <mysql-connector.version>8.0.33</mysql-connector.version>
```

Comment:
Please add this to incompatible change to the docs.

Reply:
Okay